mmv1/products/apigateway/terraform.yaml

```diff
@@ -27,15 +26,17 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         min_version: beta
         name: "apigateway_api_basic"
         primary_resource_id: "api"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-my-api%s\", context[\"random_suffix\"])"
         vars:
-          name: "api"
+          api_id: "my-api"
```

Comment:
Explanation: I updated the name of the vars because in tests where 3 names were being passed in it was easier to understand if they all had consistent names that matched the argument they were used for. For consistency I did this across all tests.

I updated the value to include a hyphen, as presence of the hyphen is needed to cause the generated test code to include `tf-test-` at the start of the value and the random suffix (from `context`) at the end.